packages/@sanity/cli/src/actions/init-project/initProject.js

```diff
@@ -31,8 +31,10 @@ export default async function initSanity(args, context) {
   const unattended = cliFlags.y || cliFlags.yes
   const print = unattended ? noop : output.print
 
-  print('This utility walks you through creating a Sanity installation.')
-  print('Press ^C at any time to quit.\n')
+  print(`This utility walks you through creating a Sanity project.
+    It installs a local instance of the Sanity studio,
+    which you can configure connects to the real-time hosted API.`)
```

Comment:
> which you can configure connects to

Missing a word here? Could also be:

> It installs a local instance of the Sanity studio, which you can configure to connect to the real-time hosted API.

or

> It installs a configurable local instance of the Sanity studio, which connects to the real-time hosted API.

Reply:
Thanks for the input @rdunk! Fixed.